database/dbscripts/addTeamAndPolicyNameConstraints.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE teams
+    ADD CONSTRAINT "Unique Team Names" UNIQUE (name, org_id);
```

Comment:
Oups, sorry to jump in, but this will be a problem in Energy Insights.

As we use nested team, this situation is legit:
- team 1
   - team A
- team 2
   - team A

What is the reason to make names unique?

Reply:
I second this ^^: there is actually nothing that comes to mind why team names should be unique. The id is unique (to be able to manage them) while the name is just for readability purposes. If we are looking at what udaru is supposed to do: ids would have been enough to be able to manage access. The name for a team (and others entities) was added as a "bonus" - bare UM functionality.

Reply:
Hi @feugy 

This is an optional script, not part of default :-)

Best wishes,

Cian

Reply:
Perfect, I wasn't aware of the optional characteristics of those scripts.
Sorry for the interruption!

Reply:
why would we commit the scripts? no point to put them in the project repo

Reply:
might be better to document that we can handle multiple constraints in readme?

This was put in based on this issue: https://github.com/nearform/udaru/issues/474

Reply:
i agree with the opinions above, let's not complicate the code with adding constraints there (even if they are on both team and parent id). If the user will add from UI two teams with the same name he can alway and edit names.